src/Acuminator/Acuminator.Utilities/Roslyn/Semantic/PXGraph/GraphSymbolUtils.cs

```diff
@@ -143,18 +143,161 @@ private static IEnumerable<INamedTypeSymbol> GetAllViewTypesFromPXGraphOrPXGraph
 				}
 			}
 		}
-		#endregion
+        #endregion
 
-		#region View Symbol And Types
-		/// <summary>
-		/// Gets all declared view symbols and types from the graph and its base graphs,
-		/// if there is a graphs class hierarchy and <paramref name="includeViewsFromInheritanceChain"/> parameter is <c>true</c>.
-		/// </summary>
-		/// <param name="graph">The graph to act on.</param>
-		/// <param name="pxContext">Context.</param>
-		/// <param name="includeViewsFromInheritanceChain">(Optional) True to include, false to exclude the views from inheritance chain.</param>
-		/// <returns/>
-		public static ViewSymbolWithTypeCollection GetViewsWithSymbolsFromPXGraph(this ITypeSymbol graph, PXContext pxContext,
+        public static IEnumerable<(MethodDeclarationSyntax Node, IMethodSymbol Symbol)> GetViewDelegatesFromGraph(
```

Comment:
@metallmonster this class became quite big. Maybe we should organaize it with regions? There are already regions for other methods? Also it's better to add xml descriptions to public helpers

Reply:
@SENya1990 I agree that is a quite big. I don't like regions because if some part of the code is not related to the other part it is better to split them via files and classes, it my opinion. I'll create a separate file.
I agree about xml descriptions.

Reply:
ok